src/librustc_trans/trans/asm.rs

```diff
@@ -164,7 +164,8 @@ pub fn trans_inline_asm<'blk, 'tcx>(bcx: Block<'blk, 'tcx>, ia: &ast::InlineAsm)
 #[cfg(any(target_arch = "arm",
           target_arch = "aarch64",
           target_arch = "mips",
-          target_arch = "mipsel"))]
+          target_arch = "mipsel",
+          target_arch = "powerpc"))]
```

Comment:
Could you rewrite this to `not(condition)` where `condition` is what's below as well?